src/css/note.css

```diff
@@ -0,0 +1,81 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css");
+
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url("https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous");
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css");
```

Comment:
you can extract this and put it in the root CSS file and then you can use it globally
Right now you are importing it on every CSS file manually and using it, which is not needed

Reply:
Ok